lib/newgistics/item.rb

```diff
@@ -5,6 +5,9 @@ class Item
     attribute :id, String
 
     attribute :sku, String
+    attribute :upc, String
```

Comment:
When are you using this field?

Reply:
I see, I just couldn't find the `upc` in the response fields listed in the Newgistics documentation, but I see it comes back as part of the response :+1: